### ISSUE TITLE
Useless error when ca-certificates are missing

### ISSUE THREAD

**Issue by glandium** [rust-lang/rust#48443]
I was trying to `cargo build` something in a newly created chroot and got the following output:
```
    Updating registry `https://github.com/rust-lang/crates.io-index`
warning: spurious network error (2 tries remaining): curl error: 
; class=Net (12)
warning: spurious network error (1 tries remaining): curl error: 
; class=Net (12)
error: failed to fetch `https://github.com/rust-lang/crates.io-index`           

Caused by:
  curl error: 
; class=Net (12)
```

Even with colors, this is not a great error message.
Strace hinted me that `/etc/ssl/certs/ca-certificates.crt` was missing, and sure enough, it worked after installing the ca-certificates package. But that wasn't thanks to cargo's error message.

**Comment by kennytm** [comment]
This is a cargo error and should be filed at https://github.com/rust-lang/cargo/issues.

**Comment by glandium** [comment]
Gah, sorry. I should have known better. Refiled as https://github.com/rust-lang/cargo/issues/5065
